datalab/datalab_session/analysis/source_catalog.py

```diff
@@ -0,0 +1,37 @@
+from datalab.datalab_session.util import get_hdu, scale_points
+import numpy as np
```

Comment:
PEP8 recommends that you separate imports into three distinct groups: https://peps.python.org/pep-0008/#imports

so it'd be something like

```
#standard library imports would go here

import numpy as np # 3rd party lib imports go here

from datalab.datalab_session.util import get_hdu, scale_points # local imports go here
```